### ISSUE TITLE
Some addon to revert all high-contrast changes

### ISSUE THREAD

**Issue by penguinmoose** [ScratchAddons/ScratchAddons#5754]
### Why this would be helpful

Some users, like me, may dislike the high-contrast changes to the Scratch website. It will be very helpful to have an addon to revert those changes.

### How this addon works

This addon changes the Scratch website style to revert the color changes for high contrast. It shouldn’t be too hard - we can simply change the CSS of the affected elements.

As far as I know, here are some high-contrast changes so far:
- View, love, favorite, and remix counters get darker
- Toggles (specifically the commenting on/off switch) get colored a darker shade of green

### Possible alternatives

We could add these features to some addon that changes the color of the website (dark-www?)

**Comment by DNin01** [comment]
We could potentially put this feature into a setting in `dark-www` called… wait, what should we call it?

**Comment by DNin01** [comment]
Also, we'd need to constantly update this as Scratch changes its costumes. At this point, I think we'd be better off allowing the community to get used to the style changes while we wait for development on the contrast improvements to conclude.

**Comment by penguinmoose** [comment]
> We could potentially put this feature into a setting in `dark-www` called… wait, what should we call it?

"revert high-contrast changes"? If we put this in `dark-www`, it might be a preset, but only if we make all the affected parts customizable.

**Comment by penguinmoose** [comment]
> Also, we'd need to constantly update this as Scratch changes different colors. At this point, I think we'd be better off allowing the community to get used to the changes while we wait for development on the contrast improvements to conclude.

That's true - should we just wait until the high-contrast changes conclude?

However, some people do want a way to revert at least some, if not all, contrast changes. We can just add a notice that it may not be totally up-to-date.

**Comment by Secret-chest** [comment]
What about customising that green colour?

**Comment by BroJac5246** [comment]
> > Also, we'd need to constantly update this as Scratch changes different colors. At this point, I think we'd be better off allowing the community to get used to the changes while we wait for development on the contrast improvements to conclude.
> 
> That's true - should we just wait until the high-contrast changes conclude?
> 
> However, some people do want a way - _quickly_ - to revert at least some, if not all, contrast changes. We can just add a notice that it may not be totally up-to-date.

That's DNin01's point- it could take months for such an addon to be added to a stable release, and, in that time, people will have gotten used to the new changes, meaning an addon for this would be pointless.

**Comment by BroJac5246** [comment]
> > > > Also, we'd need to constantly update this as Scratch changes different colors. At this point, I think we'd be better off allowing the community to get used to the changes while we wait for development on the contrast improvements to conclude.
> > > 
> > > That's true - should we just wait until the high-contrast changes conclude?
> > > 
> > > However, some people do want a way - _quickly_ - to revert at least some, if not all, contrast changes. We can just add a notice that it may not be totally up-to-date.
> > 
> > That's DNin01's point- people would want this quickly, but it could take months for such an addon to be added to a stable release, and, in that time, people will have gotten used to the new changes, meaning an addon for this would be pointless.
> 
> 2.0 style is still wanted years later, so i think it's not pointless.

But that's different. It's not a series of incredibly minor changes, but rather a complete redesign.

**Comment by DNin01** [comment]
> > > > Also, we'd need to constantly update this as Scratch changes different colors. At this point, I think we'd be better off allowing the community to get used to the changes while we wait for development on the contrast improvements to conclude.
> > > 
> > > That's true - should we just wait until the high-contrast changes conclude?
> > > 
> > > However, some people do want a way - _quickly_ - to revert at least some, if not all, contrast changes. We can just add a notice that it may not be totally up-to-date.
> > 
> > That's DNin01's point- people would want this quickly, but it could take months for such an addon to be added to a stable release, and, in that time, people will have gotten used to the new changes, meaning an addon for this would be pointless.
> 
> 2.0 style is still wanted years later, so i think it's not pointless.

The difference with offering reverting to 2.0 is that the 2.0 design will never change - it's sort of deprecated, so we'll never have to change with it, making offering a switch to go back to 2.0 easy (well, easi_er_).

The reason we haven't done that yet is because it's not high-priority. How many people want this option? Is it worth our time to spend days making an entire addon for this if only 1%* of users want 2.0 back and all those users are people who started with 2.0?

<sub>*Not meant to be an accurate representation</sub>

Also, how are 3.0 pages supposed to look in 2.0? The designs would probably be inconsistent if we tried to replicate it, especially since we're just a browser extension.

**Comment by DNin01** [comment]
> What about customising that green colour?

This is a pretty specific detail to be customizable, and you don't see it on most pages. Maybe we could put it in a group of advanced settings for that addon?

**Comment by mxmou** [comment]
> > What about customising that green colour?
> 
> This is a pretty specific detail to be customizable, and you don't see it on most pages. Maybe we could put it in a group of advanced settings for that addon?

If something like #4187 gets implemented, the green banner color could also be used for other green elements such as the remix button.

**Comment by penguinmoose** [comment]
> > > Also, we'd need to constantly update this as Scratch changes different colors. At this point, I think we'd be better off allowing the community to get used to the changes while we wait for development on the contrast improvements to conclude.
> > 
> > 
> > That's true - should we just wait until the high-contrast changes conclude?
> > However, some people do want a way - _quickly_ - to revert at least some, if not all, contrast changes. We can just add a notice that it may not be totally up-to-date.
> 
> That's DNin01's point- people would want this quickly, but it could take months for such an addon to be added to a stable release, and, in that time, people will have gotten used to the new changes, meaning an addon for this would be pointless.

Even if the high-contrast changes are going to stay, I don’t think everyone will get used to them. Some may still prefer the old colors.
